lib/fresh_connection/extend/ar_resolver.rb

```diff
@@ -14,7 +14,12 @@ def spec(*args)
           require 'fresh_connection/extend/adapters/pg_adapter'
           __extend_adapter_by_fc(::ActiveRecord::ConnectionAdapters::PostgreSQLAdapter, PgAdapter)
         else
-          raise NotImplementedError, "This adapter('#{specification.config[:adapter]}') is not supported. If you specified the mysql or postgres adapter, it's probably a bug in FreshConnection. Please teach me (https://github.com/tsukasaoishi/fresh_connection/issues/new)"
+          require 'fresh_connection/extend/adapters/other_adapter_proxy'
+          base_adapter_name = "#{specification.config[:adapter].to_s}_adapter".camelize
+          __extend_adapter_by_fc(
+            "::ActiveRecord::ConnectionAdapters::#{base_adapter_name}".constantize,
+            OtherAdapterProxy
```

Comment:
I'd like to be able to specify an extend module(like `OtherAdapterProxy`) in a configurations. 

example:
`config/initializers/fresh_connection.rb`
```ruby
FreshConnection.extend_adapter_proxy = OtherAdapterProxy
```

Could you to be able to add a function like above?
If it seems difficult for you, I will implement it.